internal/query/family_queries.go

```
@@ -174,9 +174,10 @@ func convertReadModelToFamily(rm repository.FamilyReadModel) Family {
 
 // GroupSheetEvent represents an event with date, place, and citations.
 type GroupSheetEvent struct {
-	Date      string               `json:"date,omitempty"`
-	Place     string               `json:"place,omitempty"`
-	Citations []GroupSheetCitation `json:"citations,omitempty"`
+	Date       string               `json:"date,omitempty"`
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gofmt)